sage-ios/SAGE/SAGE/ProfileCheckinViewController.swift

```diff
@@ -46,8 +47,20 @@ class ProfileCheckinViewController: SGTableViewController {
     //
     // MARK: - ViewController LifeCycle
     //
+    func setupHeader() {
+        self.tableView = UITableView(frame: self.tableView.frame, style: .Grouped)
+        self.tableView.tableHeaderView = self.userCheckinSummary
+        let headerOffset = CGFloat(100)
```

Comment:
you can refer to some constant in the userCheckinSummary instead, right? we wanna avoid repetitive hardcoding in case we wanna change the height later